<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winDataDictionary</class>
 <widget class="QMainWindow" name="winDataDictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Dictionary</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 57 12pt &quot;TerminessTTF Nerd Font&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnSave">
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Save Data Dictionary</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>Icons/Save.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAddRow">
         <property name="toolTip">
          <string>Add Row</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>Icons/Plus.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="dataDictionaryTable">
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
